[2025-02-08 19:13:20][MAIN] - Traceback (most recent call last):
[2025-02-08 19:13:20][MAIN] -   File "/home/claudio/Python-Projects/RoomServer/main.py", line 89, in <module>
[2025-02-08 19:13:20][MAIN] - main()
[2025-02-08 19:13:20][MAIN] -   File "/home/claudio/Python-Projects/RoomServer/main.py", line 77, in main
[2025-02-08 19:13:20][MAIN] - time.sleep(0.5)
[2025-02-08 19:13:20][MAIN] - KeyboardInterrupt
[2025-02-08 19:13:20][MAIN] - Contacting Arduino..
[2025-02-08 19:13:20][MAIN] - Exception ignored in: 
[2025-02-08 19:13:20][MAIN] - <module 'threading' from '/usr/lib/python3.10/threading.py'>
[2025-02-08 19:13:20][MAIN] - Traceback (most recent call last):
[2025-02-08 19:13:20][MAIN] -   File "/usr/lib/python3.10/threading.py", line 1567, in _shutdown
[2025-02-08 19:13:20][MAIN] - lock.acquire()
[2025-02-08 19:13:20][MAIN] - KeyboardInterrupt
[2025-02-08 19:13:20][MAIN] - : 
